src/nat/llm/huggingface_llm.py

```
@@ -145,8 +145,7 @@ async def huggingface_provider(
     except ImportError:
         raise ValueError("HuggingFace dependencies not installed. \n"
                          "Install with:\n"
-                         "  `pip install nvidia-nat[huggingface]` (package) or\n"
-                         "  `uv pip install -e '.[huggingface]'` (source)\n")
+                         "  `pip install \"transformers[torch,accelerate]~=4.57\"")
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)